container/image.bzl

```diff
@@ -229,6 +229,9 @@ def _impl(ctx, files=None, file_map=None, empty_files=None, directory=None,
     entrypoint: str List, overrides ctx.attr.entrypoint
     cmd: str List, overrides ctx.attr.cmd
     symlinks: str Dict, overrides ctx.attr.symlinks
+    env: str Dict overrides ctx.attr.env
+    debs: File list, overrides ctx.files.debs
+    output: File to use as output for script to load docker image
```

Comment:
only output, remove the rest of the changes

Reply:
Done.